spec/services/hyrax/workflow/deposited_notification_spec.rb

```diff
@@ -1,35 +1,49 @@
 # frozen_string_literal: true
 RSpec.describe Hyrax::Workflow::DepositedNotification do
-  let(:approver) { create(:user) }
-  let(:depositor) { create(:user) }
-  let(:to_user) { create(:user) }
-  let(:cc_user) { create(:user) }
-  let(:work) { create(:generic_work, user: depositor) }
-  let(:entity) { create(:sipity_entity, proxy_for_global_id: work.to_global_id.to_s) }
+  let(:approver) { FactoryBot.create(:user) }
+  let(:depositor) { FactoryBot.create(:user) }
+  let(:to_user) { FactoryBot.create(:user) }
+  let(:cc_user) { FactoryBot.create(:user) }
+  let(:work) { FactoryBot.create(:work, user: depositor) }
```

Comment:
Would this spec pass for a Valkyrie resource work?

Reply:
no, it fails on other test setup issues. this PR isn't intended to change the tests to valkyrie.